trunk/SUAVE/Methods/Aerodynamics/Fidelity_Zero/Drag/compressibility_drag_wing.py

```diff
@@ -1,20 +1,25 @@
 # compressibility_drag_wing.py
 # 
-# Created:  Dec 2013, SUAVE Team
-# Modified: Nov 2016, T. MacDonald
-#        
+# Created:  Your Name, Dec 2013
```

Comment:
It looks like you are using an old version of this script. The prior one had many useful edits.